packages/components/src/Skeleton/Skeleton.component.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import Icon from '../Icon';
+import theme from './Skeleton.scss';
+
+function Skeleton({ type, size, width, height, name, className }) {
+	const style = { width, height };
```

Comment:
Looks like that constant is only used in the JSX declaration bellow. It should be defined right inside the JSX `style` property.